src/main/java/com/unb/tracker/web/CourseController.java

```diff
@@ -105,14 +111,72 @@ public String postCourse(@RequestBody Course course) {
 
     @PostMapping("/course")
     public String courseSave(@ModelAttribute Course course, ModelMap map, Principal principal) {
-        LOG.info("courseSave - starting - principle.name: {}", principal.getName());
         User user = userRepository.findByUsername(principal.getName());
-        if (user == null || !user.getHasExtendedPrivileges()) {
+        if (user == null || !user.getHasExtendedPrivileges())
```

Comment:
Can you put braces on this if? (Just for safety)

Reply:
Fixed.